transforms/language/doc_chunk/python/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "dpk_doc_chunk_transform_python"
-version = "0.2.2.dev1"
+version = "0.3.0"
```

Comment:
not sure how we handle these version suffixes on transforms now.  But since you depend on a dev1 version of the core library, should this also be a dev1 suffix?

Reply:
@touma-I which version should be used? I followed your advice here, but I can also change it.

Reply:
you are ok with what you have. The release process will fix all this...